@here/harp-geojson-datasource/lib/GeoJsonDataProvider.ts

```diff
@@ -51,7 +56,22 @@ export class GeoJsonDataProvider implements DataProvider {
     }
 
     async connect(): Promise<void> {
-        await this.m_tiler.connect();
+        try {
+            await this.m_tiler.connect();
+        } catch (error) {
+            if (
+                WorkerServiceProtocol.isUnknownServiceError(error) &&
+                !missingTilerServiceInfoEmitted
```

Comment:
I get why might be nice to use to only output this message once at app init, but in more interactive apps where you might add/delete DataSources... Is this the intended behaviour? Might just log the error always.

Reply:
Note that error will be logged always, in `MapView.addDataSource` (before, it was silent).
This is indeed just for convenience, note that error is thrown anyways.

Reply:
Hmm I guess you're right, as you cannot dinamically register services in the decoders at runtime, as they have to be included in the decoder before bundling.

Reply:
Of course you can dynamically register new service types, but again this is super advanced (and not sure if makes sense) so i assume that one that does tricks like this understand consequences.